assets/src/edit-story/components/library/panes/media/mediaUtils.js

```diff
@@ -85,6 +85,140 @@ export const getResourceFromMediaPicker = (mediaPickerEl) => {
   };
 };
 
+/**
+ * Generates a virtual image to pick metadata resource
+ *
+ * @param {Object} image File object
+ * @return {Promise} Resource object
+ */
+
+const getImageMetadata = (image) => {
+  const reader = new window.FileReader();
+
+  return new Promise((resolve, reject) => {
+    reader.onload = function(e) {
+      const src = e.target.result;
+      const randomId = Math.floor(Date.now() / 1000);
+
+      const img = new window.Image();
+
+      img.onload = function() {
+        const width = img.width;
+        const height = img.height;
+
+        resolve({
+          type: 'image',
+          src,
+          width,
+          height,
+          mimeType: image.type,
+          posterId: randomId,
```

Comment:
Why is this needed? Can we remove all video attributes here instead of `undefined`/fake values?

Reply:
Sure! I will replace that to promote `undefined`.  Thanks for catching this up.